categories.xml

```diff
@@ -1,4 +1,5 @@
 <categories>
-	<category name="Uncategorized" slug="uncategorized"/>
+	<category name="Uncategorized" slug="uncategorized"></category>
+        <category name="Configuration and Utilities" slug="config"></category>
```

Comment:
> Configuration and Utilities 'cause is the standart for others jQuery docs, similar case is qUnit docs.

Definitely not a standard just because it exists in a single site. If anything, the standard would be for separate categories as other projects have a Utilities category. I'd actually say that QUnit should split their category in two, since it's almost exclusively utilities already.